packages/osc-ecommerce/app/components/Forms/ProductForm/ProductForm.tsx

```diff
@@ -134,22 +134,21 @@ export const ProductForm = forwardRef<ElementRef<'div'>, ProductFormProps>(
                                 );
                             }
 
-                            if (option.values.length > 1) {
-                                return (
-                                    <Options
-                                        // Here we're passing the searchParamsWithDefaults as part of the key so the options remount when searchParams changes
-                                        // This is needed to keep the main form and the drawer forms in sync
-                                        key={`${id}-${index}-${
-                                            option.name
-                                        }-${searchParamsWithDefaults.get(option.name)}`}
-                                        id={id}
-                                        option={option}
-                                        searchParamsWithDefaults={searchParamsWithDefaults}
-                                    />
-                                );
-                            }
-
-                            return <></>;
+                            // if (option.values.length >= 1) {
+                            return (
+                                <Options
+                                    // Here we're passing the searchParamsWithDefaults as part of the key so the options remount when searchParams changes
+                                    // This is needed to keep the main form and the drawer forms in sync
+                                    key={`${id}-${index}-${
+                                        option.name
+                                    }-${searchParamsWithDefaults.get(option.name)}`}
+                                    id={id}
+                                    option={option}
+                                    searchParamsWithDefaults={searchParamsWithDefaults}
+                                    className={option.values.length > 1 ? '' : 'u-hidden-from@all'}
```

Comment:
i would imagine we would not need to create a class for this and simply apply the `hidden` attribute?
```suggestion
                                    hidden={option.values.length <= 1 ? true : undefined}
                                    
```

Reply:
Yeah, that was my initial approach. But the inner component `RadioGroup` doesn't extend the html attributes for the `fieldset` `hidden` didn't work :/ I guess I could wrap the `Options` in a div and put it there but not keen on the extra markup for the sake of a `hidden` property :/

Reply:
ahh no worries makes sense.